scribejava-apis/src/test/java/com/github/scribejava/apis/examples/Google20Example.java

```diff
@@ -15,28 +15,31 @@
 
     private static final String NETWORK_NAME = "G+";
     private static final String PROTECTED_RESOURCE_URL = "https://www.googleapis.com/plus/v1/people/me";
-    private static final Token EMPTY_TOKEN = null;
 
-    public static void main(final String... args) {
-        // Replace these with your client id and secret
-        final String clientId = "your client id";
-        final String clientSecret = "your client secret";
-        final String secretState = "secret" + new Random().nextInt(999_999);
+    public static void main(final String[] args) {
+        final Scanner in = new Scanner(System.in, "UTF-8");
+
+        System.out.println("=== " + NETWORK_NAME + "'s OAuth Workflow ===");
+        System.out.println();
+        
+        System.out.println("Enter your Google API client key");
+        System.out.print(">>");
+        final String clientId = in.nextLine();
+        System.out.println("Enter your Google API client secret");
+        System.out.print(">>");
+        final String clientSecret = in.nextLine();
+        final String secretState = "state_" + new Random().nextInt(999_999);
         final OAuth20Service service = new ServiceBuilder()
                 .apiKey(clientId)
                 .apiSecret(clientSecret)
                 .scope("profile") // replace with desired scope
                 .state(secretState)
-                .callback("http://example.com/callback")
+                //.callback("http://example.com/callback")
```

Comment:
It was just easier to test as an OOB call. We can add the callback back if you prefer to have it that way.